third_party/nlohmann_json/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+message(STATUS "Fetching external nlohmann_json")
+include(FetchContent)
+FetchContent_Declare(
+    nlohmann_json
+    GIT_REPOSITORY https://github.com/nlohmann/json.git
+    GIT_TAG        v3.9.1
+)
+set(JSON_BuildTests OFF CACHE INTERNAL "")
```

```suggestion
set(JSON_BuildTests OFF CACHE INTERNAL "")
set(JSON_Install OFF CACHE INTERNAL "")
```
in 3.9.1, they always install it but it is not necessary

---

Great catch!